docs/quickstart/setup_quickstart.md

```diff
@@ -233,29 +239,17 @@ window of processing time, as long as words are floating in.
 
 * First of all, we use **netcat** to start local server via
 
-  {% highlight bash %}
-  $ nc -l 9000
-  {% endhighlight %}
+{% highlight bash %}
+$ nc -l 9000
+{% endhighlight %}
 
 * Submit the Flink program:
 
-  {% highlight bash %}
-  $ ./bin/flink run examples/streaming/SocketWindowWordCount.jar --port 9000
-
-  Cluster configuration: Standalone cluster with JobManager at /127.0.0.1:6123
-  Using address 127.0.0.1:6123 to connect to JobManager.
-  JobManager web interface address http://127.0.0.1:8081
-  Starting execution of program
-  Submitting job with JobID: 574a10c8debda3dccd0c78a3bde55e1b. Waiting for job completion.
-  Connected to JobManager at Actor[akka.tcp://flink@127.0.0.1:6123/user/jobmanager#297388688]
-  11/04/2016 14:04:50     Job execution switched to status RUNNING.
-  11/04/2016 14:04:50     Source: Socket Stream -> Flat Map(1/1) switched to SCHEDULED
-  11/04/2016 14:04:50     Source: Socket Stream -> Flat Map(1/1) switched to DEPLOYING
-  11/04/2016 14:04:50     Fast TumblingProcessingTimeWindows(5000) of WindowedStream.main(SocketWindowWordCount.java:79) -> Sink: Unnamed(1/1) switched to SCHEDULED
```

Comment:
is this no longer printed?

Reply:
At least when I run with commit c8fa8d025684c222582 .

These messages aren't printed in console any more, and they(not exactly the same text) are logged in file `flink-<username>-taskexecutor-0-localhost.log`.

They are printed if I use flink 1.4 latest.

Reply:
That sounds like a regression and we should probably open a JIRA.

Reply:
> You are effectively addressing FLINK-9282, could you adjust the PR title and commit message?
> (There's also another bit that needs an update, see the JIRA.

Done as suggestion.

And file a issue on https://issues.apache.org/jira/browse/FLINK-9307 about suspected regression.

Reply:
@zentol Should we await [FLINK-9307](https://issues.apache.org/jira/browse/FLINK-9307) before merge this pull request? Or just revert this section to update quickstart page partly?

Reply:
It will be quite difficult to address FLINK-9307 (and probably won't happen before 1.5), so let's merge it as it is.